crates/bindings-macro/src/lib.rs

```diff
@@ -159,6 +160,28 @@ pub fn reducer(args: StdTokenStream, item: StdTokenStream) -> StdTokenStream {
     })
 }
 
+/// It turns out to be shockingly difficult to construct an [`Attribute`].
+/// That type is not [`Parse`], instead having two distinct methods
+/// for parsing "inner" vs "outer" attributes.
+///
+/// We need this [`Attribute`] in [`table`] so that we can "pushnew" it
+/// onto the end of a list of attributes. See comments within [`table`].
+fn derive_table_helper_attr() -> Attribute {
+    let source = quote!(#[derive(spacetimedb::__TableHelper)]);
+
+    struct Wrapper(Attribute);
+
+    impl Parse for Wrapper {
+        fn parse(input: ParseStream) -> syn::Result<Self> {
+            input
+                .call(Attribute::parse_outer)
+                .map(|attrs| Wrapper(attrs.into_iter().next().unwrap()))
+        }
+    }
+
+    syn::parse2::<Wrapper>(source).unwrap().0
```

Comment:
I do believe this can just be expressed as `syn::Parser::parse2(Attribute::parse_outer, source).unwrap().into_iter().next().unwrap()`

Reply:
or `Attribute::parse_outer.parse2(source)....` if you print `syn::parse::Parser` into scope

Reply:
Neat! Applied, thanks.